src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,389 @@
++++
+title = "NAT Traversal"
+date = 2020-08-12T16:02:00+02:00
+weight = 25
++++
+
+## Basic Overview
+
+Submariner establishes the dataplane tunnels between clusters over port
+`4500/UDP` by default. This port can be customized per cluster and per gateway
+and is published as part of the `Endpoint` objects.
+
+### Public vs Private IP
+
+`Endpoint` objects publish both a `PrivateIP` and a `PublicIP`. The `PrivateIP` is the IP assigned
+to an interface on the gateway node where the `Endpoint` originated. The `PublicIP` is the source
+IP for the packets sent from the gateway to the Internet which is discovered by default via
+[ipify.org](https://ipify.org), [my-ip.io](https://my-ip.io) or [seeip.org](https://seeip.org).
```

Comment:
Does it mean that in the default configuration, we will try one after the other and use the one that is successful?
Or we just use ipify.org alone?

Reply:
I will clarify, those are fallback mechanisms, if ipify fails my-ip is tried, otherwise seeip.org

Reply:
Done